pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/CreateSegmentCommand.java

```diff
@@ -334,10 +333,10 @@ public void run() {
           for (int curr = 0; curr <= _retry; curr++) {
             try {
               SegmentGeneratorConfig config = new SegmentGeneratorConfig(segmentGeneratorConfig);
-
-              String localFile = dataFilePath.getName();
-              Path localFilePath = new Path(localFile);
-              dataDirPath.getFileSystem(new Configuration()).copyToLocalFile(dataFilePath, localFilePath);
+              URI dataFileUri = URI.create(dataFilePath);
+              String[] splits = dataFilePath.split("/");
+              String localFile = splits[splits.length - 1];
```

Comment:
Delete the local file after generating the segment? (Also recommend putting it into a temp dir)
Also, not sure if it is a good idea directly using the file name because there might be conflict for nested dir.

Reply:
create a random local directory and copy file into it, then delete the directory after segment creation